05-history.md

```diff
@@ -220,3 +220,43 @@ moving backward and forward in time becomes much easier.
 >     $ git checkout <unique ID of last commit> data_cruncher.py
 >     ~~~
 > 5. Both 2 & 4
+
+
+> ## Understanding Workflow and History {.challenge}
+>
+> What is the output of the following commands?
+>
+> ~~~ {.bash}
+> $ cd planets
+> $ nano venus.txt #input the following text: Venus is beautiful and full of love
```

Comment:
What about using:

```
$ echo "Venus is beautiful and full of love" > venus.txt
```

and 

```
$ echo "Venus is too hot to be suitable as a base" >> venus.txt
````

This way we complicate a bit the exercise (difference between `>` and `>>`) and also avoid having the comment in the command line.